lib/DxcSupport/dxcapi.use.cpp

```diff
@@ -19,7 +19,7 @@ namespace dxc {
 static void TrimEOL(_Inout_z_ char *pMsg) {
   char *pEnd = pMsg + strlen(pMsg);
   --pEnd;
-  while (pEnd > pMsg && *pEnd == '\r' || *pEnd == '\n') {
+  while (pEnd > pMsg && (*pEnd == '\r' || *pEnd == '\n')) {
```

Comment:
Note change of functionality. Previously if the character in pMsg[0] was '\n', this loop would advance past the front of the array. Extremely unlikely on Windows. Still really unlikely on Linux, but not impossible.

Reply:
:+1: